src/main/java/com/mitchej123/hodgepodge/LoadingConfig.java

```diff
@@ -273,6 +274,7 @@ public LoadingConfig(File file) {
         fixNetherLeavesFaceRendering = config.get(Category.FIXES.toString(), "fixNetherLeavesFaceRendering", true, "If fancy graphics are enabled, Nether Leaves render sides with other Nether Leaves adjacent too").getBoolean();
         fixNorthWestBias = config.get(Category.FIXES.toString(), "fixNorthWestBias", true, "Fix northwest bias on RandomPositionGenerator").getBoolean();
         fixNettyNPE = config.get(Category.FIXES.toString(), "fixNettyNPE", true, "Fix NPE in Netty's Bootstrap class").getBoolean();
+        fixNettyNPE = config.get(Category.FIXES.toString(), "fixNettyNPE", true, "Fix NPE in GuiScreen tripped by the Realms GUI").getBoolean();
```

Comment:
you copy pasted too fast, it's fixRealmsGuiNPE and how have you tried in full pack ? you shouldn't be able to compile this

Reply:
Not a compile error, but possibly a runtime error. After fixing it, I somehow got new and ~~infuriating~~ exciting errors, gonna make this a draft until I work them out